components/counter-bar/src/index.js

```diff
@@ -162,13 +175,15 @@ CounterBar.Total.propTypes = {
   score: PropTypes.number,
   scoreBackgroundColor: PropTypes.string,
   scoreDisabledBackgroundColor: PropTypes.string,
+  tag: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
```

Comment:
can this not be `PropTypes.node`?

Reply:
Yes, `propTypes.node` is fine here - committed.